src/featureMatching/getFastKeypoints.ts

```diff
@@ -56,7 +74,12 @@ export function getFastKeypoints(
   image: Image,
   options: GetFastKeypointsOptions = {},
 ): FastKeypoint[] {
-  const { fastRadius = 3 } = options;
+  const {
+    fastRadius = 3,
+    scoreAlgorithm = 'FAST',
+    normaliseScores = false,
+    harrisScoreOptions = { windowSize: 7, harrisConstant: 0.04 },
```

Comment:
```suggestion
    harrisScoreOptions,
```

We should avoid repeating default values if possible.